core/src/main/java/hudson/model/AsyncAperiodicWork.java

```diff
@@ -59,19 +100,25 @@ protected AsyncAperiodicWork(String name) {
     public final void doAperiodicRun() {
         try {
             if(thread!=null && thread.isAlive()) {
-                logger.log(Level.INFO, name+" thread is still running. Execution aborted.");
+                logger.log(getSlowLoggingLevel(), name+" thread is still running. Execution aborted.");
```

Comment:
whilst you where here changing this line is should be paramaterized logging to avoid string concat if unneeded.